modules/rostests/kmtests/ntos_mm/ZwAllocateVirtualMemory.c

```diff
@@ -109,7 +109,7 @@ SimpleErrorChecks(VOID)
 
     //BASE ADDRESS TESTS
     Base = (PVOID)0x00567A20;
-    ALLOC_MEMORY_WITH_FREE(NtCurrentProcess(), Base, 0, RegionSize, (MEM_COMMIT | MEM_RESERVE), PAGE_READWRITE, STATUS_CONFLICTING_ADDRESSES, STATUS_UNABLE_TO_DELETE_SECTION);
+    ALLOC_MEMORY_WITH_FREE(NtCurrentProcess(), Base, 0, RegionSize, (MEM_COMMIT | MEM_RESERVE), PAGE_READWRITE, STATUS_CONFLICTING_ADDRESSES, STATUS_FREE_VM_NOT_AT_BASE);
```

Comment:
I think you're likely replacing one "random" return status with another here.
What is at `0x00567A20`? We have no idea -- therefore, this test appears to be nonsensical. Same with the one below.

Reply:
looks like an attempt at "random memory pointer" :)

Reply:
> this test appears to be nonsensical. Same with the one below.

Should we simply remove these 1-2 tests? Or would you suggest other ones to replace them?
See also `CustomBaseAllocation()` which uses `0x60025000`...

Reply:
Ideally we could do something like query virtual memory to find an empty region, then use that.
`CustomBaseAllocation` could make use of the same address(es)